crates/adapters/src/format/parquet/mod.rs

```diff
@@ -246,20 +261,21 @@ fn relation_to_parquet_schema(relation: &Relation) -> Result<SerdeArrowSchema, C
             SqlType::Null => DataType::Null,
             SqlType::Binary | SqlType::Varbinary | SqlType::Interval => todo!(),
             SqlType::Array => todo!("handle array types"),
```

Comment:
actually, would be nice to convert this into an error, since this can be triggered by user input

Reply:
Or I should just implement this

Reply:
How do you represent arrays in parquet?

Reply:
```
SqlType::Array => {
                // SqlType::Array implies c.component.is_some()
                let array_component = c.component.as_ref().unwrap();
                DataType::LargeList(Arc::new(ArrowField::new_list_field(
                    columntype_to_datatype(array_component),
                    c.nullable,
                )))
            }
```